.circleci/config.yml

```diff
@@ -24,6 +24,23 @@ commands:
             git submodule sync
             git submodule update --init --recursive
 
+  format:
+    steps:
+      - run:
+          name: "Format"
+          working_directory: ~/project
+          command: |
+            cmake -P cmake/format.cmake
+            if ! git diff --exit-code
+            then
+              echo "The code has a different formatting style. Try to run: cmake -P cmake/format.cmake"
```

Comment:
```suggestion
              echo "Code formatting style is not compliant. Try to run: cmake -P cmake/format.cmake on your local dev environment and push changes"
```

Reply:
As a side note I'd point out that quick changes to source files using github's in place editing feature are likely to fail (github does not format)